soma2_msgs/SOMA2Object[] objects   ### Vector of SOMA2 ids
---
bool result
std_msgs/String[] db_ids  ### These ids are unique mongodb ids which are automatically generated by mongodb
